### PR TITLE
Add default event for removal

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

File: examples/riak/riaks.py [hunk 1]
```diff
@@ -22,12 +22,19 @@
 
 from solar.interfaces.db import get_db
 
+from functools import partial
```
Comment (pigmej) [inline examples/riak/riaks.py]: What is this for ?

File: examples/riak/riaks.py [hunk 2]
```diff
@@ -22,12 +22,19 @@
...
 from solar.events.controls import React, Dep
 from solar.events.api import add_event
 
 
 db = get_db()
 
+# use to separate riak cluster from everything else
+riak_tags = ['riak']
+# use to separate haproxy from everything else
+haproxy_tags = ['haproxy']
```
Comment (loles) [inline examples/riak/riaks.py]: You don't need this tag. All haproxy resources are already tagged with: "resources=haproxy"
https://github.com/Mirantis/solar/blob/master/resources/haproxy_service/meta.yaml#L15
https://github.com/Mirantis/solar/blob/master/resources/haproxy_service_config/meta.yaml#L23
https://github.com/Mirantis/solar/blob/master/resources/haproxy_config/meta.yaml#L21

File: examples/riak/riaks.py [hunk 3]
```diff
@@ -22,12 +22,19 @@
...
+from functools import partial
+
 from solar.events.controls import React, Dep
 from solar.events.api import add_event
 
 
 db = get_db()
 
+# use to separate riak cluster from everything else
+riak_tags = ['riak']
```
Comment (loles) [inline examples/riak/riaks.py]: We should add also correct tags to resource like in haproxy

File: examples/riak/riaks.py (hunk 1 above)
Comment (dshulyak) [inline examples/riak/riaks.py]: i just forgot about it

**Pull request merged**